apache/doris
# [test](auth)Add auth test cases

[test](auth)Add auth test cases

Supplement and modify the test cases of auth, mainly including:

1. Add whether creating roles and users with Unicode characters meets expectations.
2. Check the case sensitivity of user and role names.
3. Modify the master-slave consistency case to compare whether the permissions are consistent by using random FE nodes and master nodes, and also test whether the master-slave permissions remain consistent after granting/revoking permissions.
4. After modifying the user's grant/revoke permissions, verify whether the user's permissions meet expectations.

### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 

- Behavior changed:

- Does this need documentation?

### Check List (For Reviewer who merge this PR)
